src/webui/api/torrentscontroller.cpp

```diff
@@ -567,6 +567,7 @@ void TorrentsController::addAction()
     const TriStateBool rootFolder = parseTriStateBool(params()["root_folder"]);
     const QString savepath = params()["savepath"].trimmed();
     const QString category = params()["category"];
+    const QSet<QString> tags = List::toSet(params()["tags"].split(',', QString::SkipEmptyParts));
```

Comment:
Is `,` good to use it as separator here?

Reply:
`,` was chosen because existing `/torrents/addTags` endpoint uses `,`. 

ref: https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#add-torrent-tags